### ISSUE TITLE
Is it realistic to get a VAC ban if you launch a CSGO in the achievement manager when a real CSGO is launched?

### ISSUE THREAD

**Issue by denosee** [gibbed/SteamAchievementManager#169]
Today I wanted to open achievements in the CSGO, but I forgot to turn off the already launched CSGO, I did not open the achievements, I opened it only after the CSGO was closed. The question is, can I get VAC if I started the CSGO from the achievement manager during an already running game?

**Comment by gibbed** [comment]
I'm not Valve/Steam and can't give you an answer here.

**Comment by denosee** [comment]
Thanks for the answer! I believe that you can’t get a lock because the achievement manager does not change / read the game’s memory, so you can’t get a lock.

**Comment by michael7845** [comment]
VAC doesn't do anything that would detect this, I believe valve's anticheat started hooking OpenProcess and LoadLibrary but I'm sure that this doesn't do anything like that - it would be very easy for Valve to detect this & hand out community bans if they chose to, so you might wan to bear that in mind.

**Comment by denosee** [comment]
I am reassured that I did not run SAM while playing on a secure server. Because VAC only works on it.If you also started cs: go in SAM during the already running cs: go, would you be worried? I have paranoia that will ban the wak, and there are 900 games on my account.

**Comment by gibbed** [comment]
Pointless to worry about that, because if you were flagged for that, it's already done and there's no reversing it. It will be what it is.

**Comment by denosee** [comment]
And what i should to do?Six months have passed but there is no VAC, suddenly he will come in 5 years.

**Comment by Chaython** [comment]
CSGO had an ingame announcement of a kernel level anti cheat coming soon.
I already don't reccomend it. But def. after that.
Also people crying about bugged achievements, then discussing ways to exploit it, they didn't delete any of the exploits but when I mentioned SAM I got a community warning/comment deleted "illicit or illegal activity" warning...

**Comment by michael7845** [comment]
> CSGO had an ingame announcement of a kernel level anti cheat coming soon.

Valve did not say anything about an ring0 anticheat, they pride themselves on having an unintrusive anticheat, where did you hear that?

This isn't going to get you vac banned unless Valve changes their policy.

**Comment by denosee** [comment]
Can i ask you?Does VAC work in the CSGO main menu?

**Comment by Chaython** [comment]
‘Trusted Launch’ is an optional setting which is currently in beta. It was launched yesterday by the developers to work in conjunction with the Valve Anti-Cheat (VAC) to bolster their fight against cheaters.
Maybe I missunderstood the intrusiveness in the new beta.

**Comment by michael7845** [comment]
yeah there hasn't been anything announced about a kernel mode anticheat - the new anticheat update doesn't really stop cheaters either which is ironic since it caused more bugs for legit players than cheaters.

**Comment by rohankeluskar1** [comment]
wait, you can't get VAC banned anymore by using sam now?

**Comment by gibbed** [comment]
I have never heard of any legitimate report of a VAC ban for SAM.

**Comment by rohankeluskar1** [comment]
> I have never heard of any legitimate report of a VAC ban for SAM.

weren't people used to get vac banned if using SAM during playing on VAC secured servers?

**Comment by rohankeluskar1** [comment]
or am I having mandela effect?

**Comment by michael7845** [comment]
> or am I having mandela effect?

that has never happened from what I've read, SAM doesn't r/w any game memory.

**Comment by rohankeluskar1** [comment]
> > or am I having mandela effect?
> 
> that has never happened from what I've read, SAM doesn't r/w any game memory.

https://www.steamgifts.com/discussion/oeGWk/friend-got-vac-ban-sam
This is a 5 year old thread, from what I have heard steam vac bans users who run sam when playing on vac-protected servers. 
What my question was "Does is still do that or has it been changed?"

**Comment by michael7845** [comment]
I have used it for years while on vac secured servers, it has never done that to me - it doesn't hook/modify memory (I assume) so I don't know why vac would ban for it.

**Comment by rohankeluskar1** [comment]
idk either tbh, maybe it used to do that in previous versions.

**Comment by gibbed** [comment]
SAM has never hooked/modified either Steam or any particular game in any version.

**Comment by Joshiah12** [comment]
@denosee you get VAC?

**Comment by Joshiah12** [comment]
@michaellrowley you not getting VAC running SAM while playing on vac secured servers?
